PalasoUIWindowsForms/Widgets/TextInputBox.cs

```diff
@@ -22,6 +22,8 @@ public class TextInputBox : UserControl
 		public TextInputBox()
 		{
 			_inputBox = CreateTextBox();
+			if (_inputBox.TheControl is TextBox)
+				_inputBox.TheControl.Dock = DockStyle.Fill;	// Make the real inner box match the size of the outer virtual box.
 			this.Controls.Add(_inputBox.TheControl);
```

Comment:
Could we handle the minimumSize thing here, too...or maybe in an override of OnSizeChanged?